### ISSUE TITLE
Observables with tags have no indication if description field is empty or not

### ISSUE THREAD

**Issue by mnmnc** [TheHive-Project/TheHive#532]
### Request Type
Feature Request

### Work Environment
| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | RedHat
| TheHive version   | 3.0.6

### Problem Description
There is currently no indication in the Observables List if an observable has a non-empty / empty description field.  This is more like an enhancement then feature request.

### Steps to Reproduce
1. Add case
1. Add **first** observable with tags and no description
1. Add **second** observable with description and no tags
1. Add **third** observables with both tags and description

In the observable list there is no indication for **first** and **third** observable if there is a description value. Second observable as non-obvious indication - missing tags.

### Possible Solutions
It would be beneficial to see if there is a non-empty description of the observable in the observables list.

**Comment by Rolinh** [comment]
I concur. I believe that including the observables description on the observables list tab of a case would be a very useful improvement and make the observable description feature actually useful.

**Comment by nadouani** [comment]
This depends on how analysts use the observable `description` field, as they can put a long markdown content on it.

**Comment by Rolinh** [comment]
Do you think this is the more prevalent use of the observable's description feature? On my end, the description is usually no more than "payload delivery URL", "C2 server" and so on but I get it that every team has its own way of working.

Wouldn't it be possible to have a solution that satisfies both cases? For instance, display the observable description in the observables list by default but truncate it and display the full description only on pop-over or something?

**Comment by nadouani** [comment]
I agree with you about how you use that field, it's supposed to be juste few words providing some context to the observable.

I'll give it a try.
